test/oauth2/logout_test.dart

```diff
@@ -30,7 +28,7 @@ void main() {
     await servePackages();
     await d
         .credentialsFile(
-          globalPackageServer,
+          globalPackageServer!,
```

Comment:
I start thinking we should have made this one `late`. Most of the changes revolve around this...

Reply:
I have also had the same thought, but not entirely sure it make sense. I'll give it a shot in a separate PR